msf_updates/pose_sensor_fix.yaml

```diff
@@ -1,33 +1,72 @@
+# See the following link for more information 
+# https://github.com/ethz-asl/kalibr/wiki/IMU-Noise-Model-and-Intrinsics
+# 
+# The white noise is characterized with the continous time noise spectral density. 
+# The noise spectral density is sometime also refered to as noise density.
+# The units of the noise spectral density are:
+#  acc:  [m/s^2/sqrt(Hz)]
+#  gyro: [rad/s/sqrt(Hz)]
+# The noise spectral density can be found in the datasheet.
+# 
+# The variation of the bias is characterized as a random walk. 
+# Again see https://github.com/ethz-asl/kalibr/wiki/IMU-Noise-Model-and-Intrinsics for more information
+# The units of the random walk are:
+#  acc:  [m/s^3/sqrt(Hz)]
+#  gyro: [rad/s^2/sqrt(Hz)]
+ 
+####### ADIS
+core/core_noise_acc: 0.0022563    # [m/s^2/sqrt(Hz)] 
+core/core_noise_gyr: 0.0004       # [rad/s/sqrt(Hz)] 
 
 core/core_fixed_bias: false
+core/core_noise_accbias: 8e-5     # [m/s^3/sqrt(Hz)]
+core/core_noise_gyrbias: 3e-6     # [rad/s^2/sqrt(Hz)]
 
-#########IMU PARAMETERS#######
-####### bluebird
-core/core_noise_acc: 0.083
-core/core_noise_accbias: 0.0083
-core/core_noise_gyr: 0.0013
-core/core_noise_gyrbias: 0.00013
-####### ADIS
-#core/core_noise_acc: 0.002
-#core/core_noise_accbias: 8e-5
-#core/core_noise_gyr: 0.0004
-#core/core_noise_gyrbias: 3e-6
+####### mpu6000
+#core/core_noise_acc: 0.003924    # [m/s^2/sqrt(Hz)] mpu6000 datasheet
+#core/core_noise_gyr: 0.00008726  # [rad/s/sqrt(Hz)    mpu6000 datasheet
+
+#core/core_fixed_bias: true
+#core/core_noise_gyrbias: 0.0     # for fixed bias we do not need process noise
+#core/core_noise_accbias: 0.0     # for fixed bias we do not need process noise
 
+#######################################
+#########Pose Sensor Parameters #######
+#######################################
+pose_sensor/pose_absolute_measurements: true
+pose_sensor/pose_measurement_world_sensor: false  # selects if sensor measures its position w.r.t. world (true, e.g. Vicon) or the position of the world coordinate system w.r.t. the sensor (false, e.g. ethzasl_ptam
+pose_sensor/pose_delay: 0.02                      # [s] delay of pose sensor w.r.t. imu
+
+# for the pose sensor noise levels use the std deviation the units are
+#  position: [m]
+#  orientation: [rad]
+pose_sensor/pose_use_fixed_covariance: false
+pose_sensor/pose_noise_meas_p: 0.01              # [m]
+pose_sensor/pose_noise_meas_q: 0.02              # [rad]
+
+pose_sensor/pose_initial_scale: 1
 pose_sensor/pose_fixed_scale: false
-pose_sensor/pose_noise_scale: 0.0
-pose_sensor/pose_noise_p_wv: 0.0
-pose_sensor/pose_noise_q_wv: 0.0
-pose_sensor/pose_noise_q_ic: 0.0
+pose_sensor/pose_noise_scale: 0.0     
+
+# transformation between world frame (gravity aligned) and vision frame
+pose_sensor/pose_fixed_p_wv: false              
+pose_sensor/pose_noise_p_wv: 0.0                  
+pose_sensor/pose_fixed_q_wv: false
+pose_sensor/pose_noise_q_wv: 0.0                  
+
+# transformation between pose sensor and IMU
```

Comment:
Transformation that expresses the position and orientation of the pose-sensor w.r.t. the IMU frame of reference, expressed in the IMU frame of reference.